### DISCUSSION TITLE
Kapankah membandingkan p value thd alpha atau p value thd alpha/2

### DISCUSSION THREAD

**Discussion by Micoalgo** [teamalgoritma/community#523] [Q&A] [answered]
Pada contoh kelas PS Selasa kemarin, ada contoh Kasus 1 terkait influencer dan rata2 likes serta Kasus 2 terkait berat bersih Susu. Pada kasus 1 p value dibandingkan dg Nilai alpha sedangkan Kasus 2 nya p value dibandingkan dg alpha/2. Bagaimana ya cara membedakan penggunaan/kapan membandingkan p value thd alpha atau p value thd alpha/2 ? Terimakasih
 Kasus 1 
<img width="467" alt="image" src="https://user-images.githubusercontent.com/123436963/216511021-583b7bcd-c0f0-4fed-8c49-2fbb932cbcce.png">

Kasus 2
<img width="493" alt="image" src="https://user-images.githubusercontent.com/123436963/216511150-5dd54826-fda3-4b5c-abb6-6c1d67d86e63.png">

**Comment by kevbow** [comment, marked as answer]
Dear @Micoalgo,

Kedua hal ini dibedakan berdasarkan uji hipotesis yang diangkat. Pada kasus hipotesis satu arah kita membandingkan p-value dengan alpha sementara pada kasus hipotesis 2 arah kita menggunakan alpha/2.
Pada kasus nomor 1 kita menggunakan hipotesis satu arah yang ditunjukkan pada H1 yang digunakan terkait peningkatan likes yang didapat setelah menggunakan influencer. Hipotesis ini disebut satu arah karena pengujian yang dilakukan khusus untuk kondisi peningkatan saja.
Sementara pada kasus nomor 2 hipotesis yang diangkat pada H1 adalah apakah berat bersih susu sebanyak x atau bukan. Pengujian yang diangkat mengenai satu nilai yang sudah pasti, tidak peduli lebih besar atau lebih kecil sehingga kita harus mempertimbangkan kedua kondisi tersebut maka menggunakan nilai alpha/2 untuk melihat tail kanan dan alpha/2 untuk tail kiri.

Demikian yang dapat kami sampaikan, semoga membantu.

**Reply by Micoalgo** [reply]
Terimakasih Jawabannya
